src/commands/infra/generate.test.ts

```diff
@@ -778,3 +801,79 @@ describe("Validate backend.tfvars file", () => {
     );
   });
 });
+
+describe("test checkModuleSource function", () => {
+  it("positive test", () => {
+    let res = checkModuleSource(`source="../../azure/aks-gitops"`);
+    expect(res).toBeTruthy();
```

Comment:
Let's change this to `expect(res).toBe(true)` instead. This article is good at explaining why: https://vincenttunru.com/toBeTruthy-vs-toBe-true/

Same for the one below.

Reply:
checkModuleSource is returning boolean. using is toBeTruthy is ok.